integration/gateway/gateway_suite_test.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package gateway
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/hyperledger/fabric-protos-go/common"
+	gateway "github.com/hyperledger/fabric-protos-go/gateway"
```

Comment:
Why do we need a named import when the package is already named 'gateway'?

Reply:
No idea what happened there- probably brain freeze!